datasafe-business/src/test/java/de/adorsys/datasafe/business/impl/e2e/BaseE2ETest.java

```diff
@@ -41,14 +43,16 @@ protected void writeDataToPrivate(UserIDAuth auth, String path, String data) {
         OutputStream stream = services.privateService().write(WriteRequest.forPrivate(auth, path));
         stream.write(data.getBytes());
         stream.close();
+        log.info("File {} of user {} saved to {}", LogHelper.secure(data),
+                LogHelper.secure(auth.getUserID()), LogHelper.secure(path));
     }
 
     protected AbsoluteResourceLocation<PrivateResource> getFirstFileInPrivate(UserIDAuth inboxOwner) {
         List<AbsoluteResourceLocation<PrivateResource>> files = services.privateService().list(
                 ListRequest.forPrivate(inboxOwner, "./")
         ).collect(Collectors.toList());
 
-        log.info("{} has {} in PRIVATE", inboxOwner.getUserID().getValue(), files);
+        log.info("{} has {} in PRIVATE", LogHelper.secure(inboxOwner.getUserID().getValue()), LogHelper.secure(files));
```

Comment:
I think collection/array/list should be logged differently - there we need to process item 1 by 1, maybe it makes sense to add that?

Reply:
done